### DISCUSSION TITLE
SvelteKit deployment

### DISCUSSION THREAD

**Discussion by maksymmatviievskyi** [vercel/community#3105] [Help]
### Summary

There is an issue coming up that I don't know the resolution for:
`s.at is not a function. (In 's.at(-1)', 's.at' is undefined)`
<img width="545" alt="Знімок екрана 2023-07-03 о 12 30 51" src="https://github.com/vercel/community/assets/85802051/5f901638-6016-4375-8ab4-f2f859a3ba9e">

Initially, I thought that was the git repo problem. However, it still popped up after resetting everything and clearing the commit history. The git commit works fine, but whenever I add a new project on the Vercel dashboard, it results in a mentioned build error. 

Any ideas on how to fix it?

I have also noticed a weird feature that also appears on my other projects if I click on deployment domains rather than on normal ones.

### Steps to Reproduce

To reproduce, I go to the Vercel dashboard, navigate to any of my projects and click on the DEPLOYMENT domain e.i 
<img width="1225" alt="Знімок екрана 2023-07-03 о 12 35 29" src="https://github.com/vercel/community/assets/85802051/cc8d76a4-365e-41aa-858a-ec44768b2f30">

**Comment by amyegan** [comment]
Sorry you're having trouble with the deployment, @BohnyJ. If you share your public repo or a [minimal reproduction](https://vercel.com/support/articles/creating-a-minimal-reproducible-example), that will let us get a better look at what is happening so someone here may be able to recognize what's causing the error
